timelock-impl/src/main/java/com/palantir/atlasdb/timelock/clock/ClockSkewEvents.java

```diff
@@ -22,21 +22,26 @@
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.Histogram;
 import com.codahale.metrics.MetricRegistry;
+import com.google.common.util.concurrent.RateLimiter;
 import com.palantir.logsafe.SafeArg;
 
 public class ClockSkewEvents {
     private static final long WARN_SKEW_THRESHOLD_NANOS = 10_000_000; // 10 ms
     private static final long ERROR_SKEW_THRESHOLD_NANOS = 50_000_000; // 50 ms
 
+    private static final double SECONDS_BETWEEN_EXCEPTION_LOGS = 600; // 10 minutes
+    private static final double EXCEPTION_PERMIT_RATE = 1.0 / SECONDS_BETWEEN_EXCEPTION_LOGS;
+
     private final Logger log = LoggerFactory.getLogger(ClockSkewEvents.class);
     private final MetricRegistry metricRegistry;
 
     private final Histogram clockSkew;
     private final Counter exception;
     private final Counter clockWentBackwards;
 
-    public ClockSkewEvents(MetricRegistry metricRegistry) {
+    private final RateLimiter exceptionLoggingRateLimiter = RateLimiter.create(EXCEPTION_PERMIT_RATE);
 
+    public ClockSkewEvents(MetricRegistry metricRegistry) {
         this.metricRegistry = metricRegistry;
```

Comment:
You can probably remove this at this point. It doesn't seem to be used anymore.

Reply:
Yep, done.